packages/babel-core/src/config/files/plugins.js

```diff
@@ -99,10 +99,11 @@ function resolveStandardizedName(
   dirname: string = process.cwd(),
 ) {
   const standardizedName = standardizeName(type, name);
+  const dirnames = [dirname, process.cwd()];
 
   try {
     return require.resolve(standardizedName, {
-      paths: [dirname],
+      paths: dirnames,
```

Comment:
Unfortunately this doesn't work because we compile away this call for compatibility with older Node.js versions.

Reply:
`require.resolve` is tranpiled to an in-house polyfill: https://github.com/babel/babel/blob/b3e2bcda73dea7d68b4c82bfabb92acb11b1ed90/babel.config.js#L260-L269 so we can support Node.js 6, it will be removed in Babel 8 though.

I think we can support general `paths` by applying `M._nodeModulePaths(#).concat(#)` to every element.